OneCommunityGlobal/HighestGoodNetworkApp
# Sai deepak character countdown desc

# Description
This PR is to test a newly added feature of Character countdown for description in the add equipment form of the bmdashboard of phase 2.

## Related PRS (if any):
None, this is just a frontend PR and can be tested using the latest development branch of backend
…

## Main changes explained:
- Added the countdown for the number of characters allowing only a maximum of 150 characters.
- Made changes in the AddTypeForm.jsx in BMDashboard folder under add under equipment
- added a CSS class in BMDashboard.css file
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin or owner user
5. navigate to /bmdashboard and login again and after logging in navigate to /bmdashboard/equipment/add
6. Test the working and accuracy of the character countdown added below the description text area. Verify that the color of the text turns to red when inputting more than 150 characters and changes back to black when less than 150 characters.

## Screenshots or videos of changes:
![Screenshot (100)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114311909/a58b3cbb-0fb8-4a91-9d4c-58af7dac50b9)
![Screenshot (99)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114311909/592c4306-687d-4601-8d7a-2baef4a697da)
